Add links to libclang_rt.osx.a to allow finding the emutls object that are 
used to make emulated-tls work 

--- libcxxabi-5.0.1.src/lib/buildit.orig	2018-12-08 15:14:13.000000000 -0800
+++ libcxxabi-5.0.1.src/lib/buildit	2018-12-08 15:14:52.000000000 -0800
@@ -53,7 +53,7 @@
         -compatibility_version 1 \
         -install_name /usr/lib/libc++abi.dylib \
         -Wl,-undefined,dynamic_lookup \
-        -lSystem"
+        -lSystem -L@@prefix@@/libexec/llvm-5.0/lib/clang/5.0.1/lib/darwin -lclang_rt.osx"
 	if [ -f "${SDKROOT}/usr/local/lib/libCrashReporterClient.a" ]
 	then
 		LDSHARED_FLAGS+=" -lCrashReporterClient"
